docs/reference/data/widgets_by_platform.csv

```diff
@@ -5,7 +5,7 @@ MainWindow,Core Component,:class:`~toga.app.MainWindow`,Main window of the appli
 ActivityIndicator,Core Component,:class:`~toga.widgets.activityindicator.ActivityIndicator`,A spinning activity animation,|y|,|y|,,,,
 Button,General Widget,:class:`~toga.widgets.button.Button`,Basic clickable Button,|y|,|y|,|y|,|y|,|y|,|y|
 Canvas,General Widget,:class:`~toga.widgets.canvas.Canvas`,Area you can draw on,|y|,|y|,|y|,|y|,,
-DatePicker,General Widget,:class:`~toga.widgets.datepicker.DatePicker`,An input for dates,,,|y|,,,
+DatePicker,General Widget,:class:`~toga.widgets.datepicker.DatePicker`,An input for dates,,,|y|,,|y|,
```

Comment:
There should also be a metadata entry for TimePicker

Reply:
This is done